datamad2/migrations/0036_auto_20200923_1253.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.2.13 on 2020-09-23 11:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('datamad2', '0035_merge_20200923_1253'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='jiraissuetype',
+            name='end_date_field',
+            field=models.CharField(blank=True, help_text='Format: customfield_{{number}}', max_length=100, verbose_name='End Date Field ID'),
+        ),
+        migrations.AlterField(
+            model_name='jiraissuetype',
+            name='grant_ref_field',
+            field=models.CharField(blank=True, help_text='Format: customfield_{{number}}', max_length=100, verbose_name='Grant Ref Field ID'),
+        ),
+        migrations.AlterField(
+            model_name='jiraissuetype',
+            name='issuetype',
+            field=models.IntegerField(blank=True, help_text='JIRA Data Management issue type ID. e.g. 10602', null=True, verbose_name='Issue Type Id'),
+        ),
+        migrations.AlterField(
+            model_name='jiraissuetype',
+            name='pi_field',
+            field=models.CharField(blank=True, help_text='Format: customfield_{{number}}', max_length=100, verbose_name='Principle Investigator Field ID'),
+        ),
+        migrations.AlterField(
+            model_name='jiraissuetype',
+            name='primary_datacentre_field',
+            field=models.CharField(blank=True, help_text='Format: customfield_{{number}}', max_length=100, verbose_name='Primary Datacentre Field ID'),
+        ),
+        migrations.AlterField(
+            model_name='jiraissuetype',
+            name='research_org_field',
+            field=models.CharField(blank=True, help_text='Format: customfield_{{number}}', max_length=100, verbose_name='Research Org Field ID'),
+        ),
+        migrations.AlterField(
+            model_name='user',
+            name='is_admin',
+            field=models.BooleanField(default=False, verbose_name='Give admin status'),
+        ),
+    ]
```

Comment:
It would be good to clean up the migrations. There is a different 032 migration in master.
1. migrate back to 0031_auto_20200917_1424.py using `python manage.py migrate datamad2 0031_auto_20200917_1424`
2. delete migrations 0032 - end
3. pull and merge the master branch into this one to pickup 032 migration
4. run `python manage.py make migrations`

This should then result in just one. 033. 🤞

Reply:
Think I've done this.